contentcuration/contentcuration/static/js/edit_channel/archive/views.js

```diff
@@ -21,15 +21,16 @@ var ArchiveModalView = BaseViews.BaseModalView.extend({
     template: require("./hbtemplates/archive_modal.handlebars"),
     name: NAMESPACE,
     $trs: MESSAGES,
-
+    modal: true,
     initialize: function(options) {
-        this.modal = true;
+        _.bindAll(this, "closed_modal")
```

Comment:
Not at all a blocker, but if you foresee more functions needing to be bound like this across multiple modals, you could also create a  method on the base class to take care of this (e.g. [see here](https://github.com/learningequality/studio/blob/develop/contentcuration/contentcuration/static/js/edit_channel/views.js#L572))

Reply:
Did you merge this pr into develop? The first three are issues that were directly pushed to staging I think, so develop wouldn't have those fixes. Maybe we could quickly merge in the "There and back again" PRs tomorrow to sync everything up?